### PR TITLE
Add LVGL tutorial

### PR DESCRIPTION
**Pull request by baylessj**
Putting too much information in an LVGL tutorial is redundant with their own docs, but it's probably worth adding a tutorial for it to help point users towards the LVGL site (compared to the single link in the extended API currently), since so many users are asking about it.

I think the main goal is to help redirect users to LVGL's site as opposed to explaining the ins and outs of the library ourselves, but I'm definitely open to suggestions on that.

### CODE REVIEW

**Comment by Octogonapus** [comment]
I think there should be emphasis that users don't need to initialize LVGL, there has been a lot of confusion around that.

File: v5/tutorials/topical/display.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+LVGL is a full-featured C graphics library (it's accessible in C++ projects too under the same API).
+
+The first step to getting started with LVGL is to include ``apix.h`` in your ``main.h`` file or other header files.
+This includes the full LVGL feature set as described in their documentation: https://littlevgl.com/basics
+
+From there, you can follow along with any of the LVGL tutorials (such as the one linked above). There is no need
+to initialize LVGL, you can simply start creating objects.
+
+.. warning:: LVGL code should be executed in one of the main PROS tasks (``initialize()``, ``autonomous()``,
+             or ``opcontrol()``). Running LVGL code in a user-created task will fail to change the display.
```
Comment (HotelCalifornia) [inline v5/tutorials/topical/display.rst]: nope, this is wrong. it was determined that the issues reported regarding this were caused by starving the LVGL daemon (i.e. by not adding a delay in a task)

Comment (HotelCalifornia) [inline v5/tutorials/topical/display.rst]: Also, we should make a note that LLEMU cannot be run at the same time as custom LVGL code

Comment (baylessj) [reply v5/tutorials/topical/display.rst]: Gotcha, I missed that in my skimming of the conversation. I'll update the tutorial accordingly.

File: v5/tutorials/topical/display.rst [hunk 2]
```diff
@@ -11,5 +11,4 @@ This includes the full LVGL feature set as described in their documentation: htt
 From there, you can follow along with any of the LVGL tutorials (such as the one linked above). There is no need
 to initialize LVGL, you can simply start creating objects.
 
-.. warning:: LVGL code should be executed in one of the main PROS tasks (``initialize()``, ``autonomous()``,
-             or ``opcontrol()``). Running LVGL code in a user-created task will fail to change the display.
\ No newline at end of file
+.. note:: Custom LVGL code cannot be displayed at the same time as the `LLEMU <./llemu.html>`_.
```
Comment (HotelCalifornia) [inline v5/tutorials/topical/display.rst]: hm, we're getting there. can you mention somewhere specifically that if people want to use custom LVGL code, they'll have to get rid of the stuff that's there by default in `initialize.cpp`?

Comment (baylessj) [reply v5/tutorials/topical/display.rst]: Right, forgot about the llemu code that's in there by default. Will do.

File: v5/tutorials/topical/display.rst [hunk 3]
```diff
@@ -0,0 +1,16 @@
+=======================
+V5 Brain Display (LVGL)
+=======================
+
+Interacting with the touchscreen on the V5 Brain is made possible through `LVGL <https://littlevgl.com>`_.
+LVGL is a full-featured C graphics library (it's accessible in C++ projects too under the same API).
+
+The first step to getting started with LVGL is to include ``apix.h`` in your ``main.h`` file or other header files.
+This includes the full LVGL feature set as described in their documentation: https://littlevgl.com/basics
```
Comment (HotelCalifornia) [inline v5/tutorials/topical/display.rst]: Does this link end up as a hyperlink?

Comment (baylessj) [reply v5/tutorials/topical/display.rst]: yep! Anything starting with `http(s)://` gets turned into a link by Sphinx

**Pull request merged**